lib/index.js

```diff
@@ -337,7 +337,7 @@ module.exports = (bookshelf) => {
                         if (err.message && err.message === 'EmptyResponse') {
                             return reject(this.NOT_FOUND);
                         }
-                        if (err.code) {
+                        if (err.code && err.code !== 'ER_SIGNAL_EXCEPTION') {
```

Comment:
Why exactly this error code?

Reply:
I don't want to break backward compatibility - but I'm happy to include others or change this?
I was thinking mostly for this PR though:
https://github.com/betastreet/exchange-api-integrations/pull/79